megamek/src/megamek/common/units/LandAirMek.java

```diff
@@ -1559,6 +1551,22 @@ public HitData rollHitLocation(int table, int side) {
             }
         }
 
+        boolean playtestLocations = gameOptions().booleanOption(OptionsConstants.PLAYTEST_1);
```

Comment:
The three implementations of this code are nearly identical.
Please refactor into a single function that is called from each class's `rollHitLocation()`